cmd/start.go

```diff
@@ -92,17 +93,22 @@ func Start(c *cli.Context) error {
 		return err
 	}
 
-	chainProvider, err := ethereum_v1.Connect(ctx, config.Ethereum)
+	beaconChain, _, err := ethereum.Connect(ctx, config.Ethereum)
 	if err != nil {
 		return fmt.Errorf("error connecting to Ethereum node: [%v]", err)
 	}
 
-	blockCounter, err := chainProvider.BlockCounter()
+	legacyChainProvider, err := ethereum_v1.Connect(ctx, config.Ethereum)
```

Comment:
s/`legacyChainProvider`/`chainProviderV1`? Should make more sense as we have `ethereum_v1` package. This will also allow to avoid conflicts with https://github.com/keep-network/keep-core/pull/3076.